test-packages/release/src/publish.ts

```diff
@@ -104,9 +195,10 @@ To publish a release you should start from a clean repo. Run "embroider-release
   let { solution, description } = loadSolution();
 
   if (!process.env.GITHUB_AUTH) {
-    process.stderr.write(`You need to set GITHUB_AUTH.`);
+    process.stderr.write(`\nYou need to set GITHUB_AUTH.`);
```

Comment:
also why are we prefixing everything with `\n` instead of just making sure that everything ends with an `\n`?

Reply:
So that it stands out -- error messages are important, and we aren't giving them any color or anything, so they need another means of standing out.